From 9c07a14529348711e36809ce71a063e0a8910e09 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Tue, 18 Mar 2014 09:18:50 -0700
Subject: [PATCH 494/827] media: xilinx: tpg: Use xvip_start()

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Use xvip_start().

Signed-off-by: Hyun Kwon <hyunk@xilinx.com>
Acked-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit ecf0cefd6fdcaed8b8bcd190add28f98e7d250cc)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-tpg.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-tpg.c b/drivers/media/platform/xilinx/xilinx-tpg.c
index 9b4dcc8..cf5b206 100644
--- a/drivers/media/platform/xilinx/xilinx-tpg.c
+++ b/drivers/media/platform/xilinx/xilinx-tpg.c
@@ -144,8 +144,7 @@ static int xtpg_s_stream(struct v4l2_subdev *subdev, int enable)
 		   (height << XVIP_ACTIVE_VSIZE_SHIFT) |
 		   (width << XVIP_ACTIVE_HSIZE_SHIFT));
 
-	xvip_write(&xtpg->xvip, XVIP_CTRL_CONTROL, XVIP_CTRL_CONTROL_SW_ENABLE |
-		   XVIP_CTRL_CONTROL_REG_UPDATE);
+	xvip_start(&xtpg->xvip);
 
 	return 0;
 }
-- 
2.9.3

